### PR TITLE
Fix #89: Add `orientation` to `window.screen`

### LINKED ISSUE #89: Is there an object for screen orientation?
Hello! I'm trying to add [screen orientation event listener](https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation/onchange) but I couldn't find any object like `screen.orientation` in the library. Is it available in this library with a different name or in some other library (like Feliz)?

### PR DESCRIPTION
**Pull request by sumeetdas**
This PR will add [`orientation`](https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation) to `window.screen` object.

### CODE REVIEW

File: src/Dom/Browser.Dom.fs [hunk 1]
```diff
@@ -808,6 +808,32 @@ type [<AllowNullLiteral>] TreeWalkerType =
 type [<AllowNullLiteral>] BarProp =
     abstract visible: bool with get, set
 
+[<StringEnum; RequireQualifiedAccess>]
```
Comment (alfonsogarciacaro) [inline src/Dom/Browser.Dom.fs:811]: If you use `StringEnum(CaseRules.KebabCase)` you don't need to use `CompiledName` in every case :)

File: src/Dom/Browser.Dom.fs [hunk 2]
```diff
@@ -808,6 +808,32 @@ type [<AllowNullLiteral>] TreeWalkerType =
...
+    | [<CompiledName("portrait")>] Portrait
+    | [<CompiledName("portrait-primary")>] PortraitPrimary
+    | [<CompiledName("portrait-secondary")>] PortraitSecondary
+    | [<CompiledName("landscape-primary")>] LandscapePrimary
+    | [<CompiledName("landscape-secondary")>] LandscapeSecondary
+
+type [<AllowNullLiteral>] ScreenOrientation =
+    inherit EventTarget
+    abstract ``type``: ScreenOrientationType
+    abstract angle: int
```
Comment (alfonsogarciacaro) [inline src/Dom/Browser.Dom.fs:832]: According to MDN this is an unsigned short, but `int` is probably more convenient :+1:

File: src/Dom/Browser.Dom.fsproj [hunk 3]
```diff
@@ -2,8 +2,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <PackageId>Fable.Browser.Dom</PackageId>
-    <Version>2.7.0</Version>
-    <PackageVersion>2.7.0</PackageVersion>
+    <Version>2.8.0</Version>
+    <PackageVersion>2.8.0</PackageVersion>
```
Comment (alfonsogarciacaro) [inline src/Dom/Browser.Dom.fsproj:6]: You don't need to bump the version here. This is automatically done by the build script :)

**Review by alfonsogarciacaro — APPROVED** [review]
Thanks a lot @sumeetdas! I left a couple of comments but the PR looks good to merge!

File: src/Dom/Browser.Dom.fs (hunk 1 above)
Comment (alfonsogarciacaro) [reply src/Dom/Browser.Dom.fs:811]: Ups, actually I just realized that Browser.Dom was using an old Fable.Core version that didn't allow this, I've bumped the version 👍

File: src/Dom/Browser.Dom.fs (hunk 2 above)
Comment (sumeetdas) [reply src/Dom/Browser.Dom.fs:832]: Sorry, I didn't know any equivalent of `ushort` in F#, so went with `int`

File: src/Dom/Browser.Dom.fsproj (hunk 3 above)
Comment (sumeetdas) [reply src/Dom/Browser.Dom.fsproj:6]: Got it! 👍

**Pull request merged**